tools/setup/emoji/generate_emoji_names

```diff
@@ -72,9 +108,47 @@ def main() -> None:
             all_canonical_names.add(canonical_name)
             all_emojis[emoji_code] = CUSTOM_EMOJI_NAME_MAPS[emoji_code]
         else:
-            continue  # this commit doesn't add CLDR data yet.
             # create the unicode character(s) for the emoji, since this is the key into the CLDR data
+            emoji = "".join(
+                chr(int(h, 16))
+                for h in (emoji_dict["non_qualified"] or emoji_dict["unified"]).split("-")
+            )
+            if emoji not in CLDR_DATA:
+                print(
+                    f"{emoji} not found in custom emoji name maps, but also not found in CLDR data. Skipping."
+                )
+                continue
+            assert len(CLDR_DATA[emoji]["tts"]) == 1
```

Comment:
Can you add a comment for what this assertion is doing? I have no idea what `tts` means. 

Another idea that might be better is to add a comment with a link to the part of the documentation for the CLDR data library defining its format; or if that doesn't exist, just give an example shape in a comment.

Would make this a lot easier to read :)

Reply:
The documentation is in [Unicode Technical Standard #35](https://www.unicode.org/reports/tr35/tr35-general.html#14-annotations-and-labels).

Reply:
Cool, let's add a link to that standard in the comment as well.